### ISSUE TITLE
Proper ECONNRESET handling for global connection

### ISSUE THREAD

**Issue by baoshan** [patriksimek/co-mssql#1]
Hi,

```
sql = require 'co-mssql`
yield sql.connect config
sql.on 'error', (err) -> console.error err
```

Prompts: `sql.on` is not a function (`undefined` actually).

The app occasionally crashes due to
```
events.js:141
      throw er; // Unhandled 'error' event
      ^
ConnectionError: Connection lost - read ECONNRESET
```

Please advise, thanks.

**Comment by patriksimek** [comment]
Fixed in 1.2.1. Just update your co-mssql.

**Comment by baoshan** [comment]
Cool! Have a nice day~
